src/lib.rs

```diff
@@ -773,8 +772,12 @@ impl<S: VhostUserBackend> VhostUserSlaveReqHandlerMut for VhostUserHandler<S> {
             // Close file descriptor set by previous operations.
             let _ = unsafe { libc::close(kick.as_raw_fd()) };
```

Comment:
This can be simply removed. It's not needed since `self.vrings.kick` is an `EventFd`, and as soon as it is replaced with another EventFd, the former will go out of scope and the fd will be automatically closed.

Reply:
I think you're right here as well, but wasn't broken even before?

In any case, happy to fix it while we're at it.

Reply:
Oh yes this wasn't needed even before. You can use a separate commit to fix this issue only, since it's not related to the API change.

Reply:
Will do.